### PR TITLE
Ensure the board expression indexes are in the schema.rb

### PR DESCRIPTION
**Pull request by camallen**
Moved from the postgres 9.5 `if exists` sql syntax to rails index helpers and in the process preserved the expression indexes in the schema.rb via the schema_plus_pg_indexes gem. It was that or move to a sql dump format and modify the test setup scripts, etc.

Unfortunately it modified the schema file format which is a pretty verbose diff :( but we now have those indexes preserved in the schema.rb file :)

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -152,208 +133,159 @@
...
+  add_index "user_conversations", ["conversation_id", "user_id"], :name=>"index_user_conversations_on_conversation_id_and_user_id"
 
   create_table "user_ip_bans", force: :cascade do |t|
-    t.cidr     "ip",         null: false
+    t.cidr     "ip",         :null=>false, :index=>{:name=>"index_user_ip_bans_on_ip"}
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  add_index "user_ip_bans", ["ip"], name: "index_user_ip_bans_on_ip", using: :btree
```
Comment (marten) [inline db/schema.rb]: Why did this disappear

Comment (camallen) [inline db/schema.rb]: it went into the create_table definition

**Comment by camallen** [comment]
I'm happy enough to switch to dumping schema in sql format ala panoptes and updating the docs / rake tasks, etc. FYI - we are using these schema_plus gems in panoptes as well.

**Comment by marten** [comment]
Ahh, I get it now. No, I have the structure.sql more than I hate this.

**Pull request merged**